app/controllers/guests_controller.rb

```diff
@@ -0,0 +1,13 @@
+class GuestsController < ApplicationController
+  
+  def show
+    @guest = Guest.find(params[:id])
+  end
+
+  def add_room
+    @guest = Guest.find(params[:id])
```

Comment:
Nice work getting this data back from the form! A more RESTful option here though would be to direct that form from the show page back to a different action, and specifically to a create action in a joins controller because:

- We will be associating an existing room to an existing guest, thereby creating a joins record
- we don't want a create action in this controller because that should be reserved for if you ever wanted to create a Guest resource. In this case we're associating an existing guest to a room which is different. That's why we could make use of a joins controller called a GuestRoomsController and put the create action there. Once the joins record is created, you can redirect back to "/guests/:id" to see the show page again